godot-codegen/src/util.rs

```diff
@@ -55,6 +55,14 @@ pub fn make_sname_ptr(identifier: &str) -> TokenStream {
     }
 }
 
+fn class_is_server(class_name: &str) -> bool {
+    class_name
+        .trim_end_matches("Extension")
+        .trim_end_matches("2D")
+        .trim_end_matches("3D")
```

Comment:
This is incorrect. Example: `Node2D`.

Maybe better to list the classes explicitly.

Reply:
`trim_end_matches` just removes something from the. so this entire statement really is like matching a regex
`.*Server(2D|3D)?(Extension)?`
so `Node2D` wouldn't actually match here